io-engine/src/bdev/nexus/nexus_bdev_children.rs

```diff
@@ -1068,7 +1069,7 @@ impl<'n> Nexus<'n> {
         // Cancel rebuild job for this child, if any.
         if let Some(job) = child.rebuild_job() {
             debug!("{self:?}: retire: stopping rebuild job...");
-            let terminated = job.terminate();
+            let terminated = job.terminate(RebuildOperation::Fail);
```

Comment:
would it be better to add a fail method to avoid having to pass operation and have assert inside?

Reply:
@dsavitskiy wdyt?

Reply:
ok, will change